ffi/gn/.gitignore

```diff
@@ -1,3 +1,6 @@
 out/
 third_party_tools/
 vendor/
+.cargo/config
+icu4x/BUILD.gn
+Cargo.lock
```

Comment:
question: the file is gitignored but still checked in to git. intentional?

(I can see wanting to do this for a file that you want to "seed" but don't care about diffs for, and will require an explicit git add to modify. which seems appropriate-ish)

Reply:
It was supposed to be deleted. Fixed